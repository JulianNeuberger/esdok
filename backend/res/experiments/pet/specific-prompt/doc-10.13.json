{"text": "The INQ transmits the transaction data request to the IP . The IP checks the request of the INQ . The IP answers the question of the INQ depending on the outcome of the examination , i.e . Transmission of data or rejection .", "name": "doc-10.13", "id": "doc-10.13", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "INQ", "indexInDocument": 1, "posTag": "NNP", "sentenceIndex": 0}, {"text": "transmits", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 3, "posTag": "DT", "sentenceIndex": 0}, {"text": "transaction", "indexInDocument": 4, "posTag": "NN", "sentenceIndex": 0}, {"text": "data", "indexInDocument": 5, "posTag": "NNS", "sentenceIndex": 0}, {"text": "request", "indexInDocument": 6, "posTag": "NN", "sentenceIndex": 0}, {"text": "to", "indexInDocument": 7, "posTag": "TO", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 8, "posTag": "DT", "sentenceIndex": 0}, {"text": "IP", "indexInDocument": 9, "posTag": "NNP", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 10, "posTag": ".", "sentenceIndex": 0}, {"text": "The", "indexInDocument": 11, "posTag": "DT", "sentenceIndex": 1}, {"text": "IP", "indexInDocument": 12, "posTag": "NNP", "sentenceIndex": 1}, {"text": "checks", "indexInDocument": 13, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 14, "posTag": "DT", "sentenceIndex": 1}, {"text": "request", "indexInDocument": 15, "posTag": "NN", "sentenceIndex": 1}, {"text": "of", "indexInDocument": 16, "posTag": "IN", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 17, "posTag": "DT", "sentenceIndex": 1}, {"text": "INQ", "indexInDocument": 18, "posTag": "NNP", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 19, "posTag": ".", "sentenceIndex": 1}, {"text": "The", "indexInDocument": 20, "posTag": "DT", "sentenceIndex": 2}, {"text": "IP", "indexInDocument": 21, "posTag": "NNP", "sentenceIndex": 2}, {"text": "answers", "indexInDocument": 22, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 23, "posTag": "DT", "sentenceIndex": 2}, {"text": "question", "indexInDocument": 24, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 25, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 26, "posTag": "DT", "sentenceIndex": 2}, {"text": "INQ", "indexInDocument": 27, "posTag": "NNP", "sentenceIndex": 2}, {"text": "depending", "indexInDocument": 28, "posTag": "VBG", "sentenceIndex": 2}, {"text": "on", "indexInDocument": 29, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 30, "posTag": "DT", "sentenceIndex": 2}, {"text": "outcome", "indexInDocument": 31, "posTag": "NN", "sentenceIndex": 2}, {"text": "of", "indexInDocument": 32, "posTag": "IN", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 33, "posTag": "DT", "sentenceIndex": 2}, {"text": "examination", "indexInDocument": 34, "posTag": "NN", "sentenceIndex": 2}, {"text": ",", "indexInDocument": 35, "posTag": ",", "sentenceIndex": 2}, {"text": "i.e", "indexInDocument": 36, "posTag": "NN", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 37, "posTag": ".", "sentenceIndex": 2}, {"text": "Transmission", "indexInDocument": 38, "posTag": "NN", "sentenceIndex": 3}, {"text": "of", "indexInDocument": 39, "posTag": "IN", "sentenceIndex": 3}, {"text": "data", "indexInDocument": 40, "posTag": "NNS", "sentenceIndex": 3}, {"text": "or", "indexInDocument": 41, "posTag": "CC", "sentenceIndex": 3}, {"text": "rejection", "indexInDocument": 42, "posTag": "NN", "sentenceIndex": 3}, {"text": ".", "indexInDocument": 43, "posTag": ".", "sentenceIndex": 3}], "mentions": [{"type": "activity", "tokenDocumentIndices": [2, 3, 4, 5, 6]}, {"type": "activity", "tokenDocumentIndices": [13, 14, 15]}, {"type": "activity", "tokenDocumentIndices": [22, 23, 24]}, {"type": "activity", "tokenDocumentIndices": [38, 39, 40]}, {"type": "activity", "tokenDocumentIndices": [42]}, {"type": "actor", "tokenDocumentIndices": [0, 1]}, {"type": "actor", "tokenDocumentIndices": [11, 12]}, {"type": "actor", "tokenDocumentIndices": [20, 21]}, {"type": "activity data", "tokenDocumentIndices": [3, 4, 5, 6]}, {"type": "activity data", "tokenDocumentIndices": [14, 15]}, {"type": "activity data", "tokenDocumentIndices": [23, 24]}, {"type": "activity data", "tokenDocumentIndices": [40]}, {"type": "further specification", "tokenDocumentIndices": [28, 29, 30, 31, 32, 33, 34]}, {"type": "xor gateway", "tokenDocumentIndices": [41]}, {"type": "condition specification", "tokenDocumentIndices": [41]}], "entities": [{"mentionIndices": []}, {"mentionIndices": [6, 7]}, {"mentionIndices": [8, 9]}, {"mentionIndices": []}, {"mentionIndices": []}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [5]}, {"mentionIndices": [10]}, {"mentionIndices": [11]}, {"mentionIndices": [12]}, {"mentionIndices": [13]}, {"mentionIndices": [14]}], "relations": [{"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 13, "type": "flow"}, {"headMentionIndex": 13, "tailMentionIndex": 14, "type": "flow"}, {"headMentionIndex": 14, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 14, "tailMentionIndex": 4, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 5, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 8, "type": "uses"}, {"headMentionIndex": 1, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 9, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 7, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 10, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 12, "type": "further specification"}, {"headMentionIndex": 3, "tailMentionIndex": 11, "type": "uses"}]}