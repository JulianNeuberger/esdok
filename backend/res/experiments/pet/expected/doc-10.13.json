{"nodes": [{"id": "0", "name": "The INQ", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "transmits", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "the request of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "answers", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "the question of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "depending on the outcome of the examination", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "1", "name": "transmits", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "the request of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "1", "name": "transmits", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "2", "source": {"id": "1", "name": "transmits", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "0", "name": "The INQ", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "3", "source": {"id": "1", "name": "transmits", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "4", "source": {"id": "3", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "the request of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "3", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "3", "name": "checks", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "answers", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "7", "source": {"id": "5", "name": "answers", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "the question of the INQ", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "8", "source": {"id": "5", "name": "answers", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the IP", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "9", "source": {"id": "5", "name": "answers", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "depending on the outcome of the examination", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-10.13", "pageStart": 1, "pageEnd": 1}}, "type": "further specification"}]}